packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/service/ServiceEditorState.ts

```diff
@@ -101,6 +110,22 @@ export const isServiceQueryTDS = (
 };
 
 export const MINIMUM_SERVICE_OWNERS = 2;
+export const OWNERSHIP_OPTIONS = [
+  {
+    label: 'Deployment',
+    value: ServiceOwnershipType.DEPLOYMENT_OWNERSHIP,
+  },
+  {
+    label: 'User List',
+    value: ServiceOwnershipType.USERLIST_OWNERSHIP, //"UserListOwnership",
+  },
+];
+
+export type ServiceOwnerOption = {
```

Comment:
maybe just to be consistent. is it better to move it to ServiceEditor.tsx under line 
`type UserOption = { label: string; value: string };`

Reply:
the type is used in a function in editor state too so can't move it to the editor

Reply:
you could export it